rule-engine/rule-engine-demos-test/.classpath

```diff
@@ -36,5 +36,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="con" path="RULE_ENGINE_CONTAINER"/>
```

Comment:
shouldn't we better put this into the 'rule-engine-demos' itself? then every workspace depending on it would work :thinking: